src/contracts/libraries/SlashingLib.sol

```diff
@@ -141,7 +149,18 @@ library SlashingLib {
             .mulWad(slashingFactor);
     }
 
-    function calcSlashedAmount(uint256 operatorShares, uint256 wadSlashed) internal pure returns (uint256) {
-        return operatorShares.mulWad(wadSlashed);
+    function calcSlashedAmount(
+        uint256 operatorShares,
+        uint256 queuedWithdrawnScaledShares,
```

Comment:
`scaledSharesInQueue` - shorter

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/881/commits/173c411ca2bc66853c4f417d09557a35dc5a4935 `slashableSharesInQueue` since the function just takes in the shares after 'descaling' the scaled shares